script.js

```diff
@@ -29,10 +30,10 @@ const getBreedData = async () => {
         breedsData[i].image.url
       }" alt="Image of the ${breedsData[i].name}">`;
       resultData.append(newLi);
-      breedImages = document.querySelectorAll(".hidden");
-      newLi.addEventListener("click", function () {
-        breedImages[i].classList.toggle("result__breedImage");
-      });
+    }
+
+    for (let i = 0; i < allLi.length; i++) {
```

Comment:
huh.. ne to delat, da na velik elementov daješ event listenerje. Event listenerje je treba minimizirat, jih imet čimmanj kot je le možno. 
Za ta primer recimo, daj listener na nek container, ki zawrapa vse te elemente in poslušaš na klike tam in pol glede na id ali pa nek identifier, veš ker se je kliknu.
Članek o tem: https://javascript.info/bubbling-and-capturing
če boš hotu še kaj o tem zvedt pogooglej `event bubbling` ali pa `event propagation`

Reply:
Ok, make sense  :) Popravim kodo